org/Konsgn/index.md

```diff
@@ -0,0 +1,5 @@
+---
+layout: org
+title: Konsgn
+---
+Just want a stable pid for a universal bootloader. Woo!
```

Comment:
Better to leave this out if you don't have a good description for your org. Other than that, looks good!